lib/inquirer-prompt.js

```diff
@@ -18,11 +18,14 @@ module.exports = function prompt(questions, config) {
 	.reduce(function (newOpts, ans) {
 		return attachAnswers(newOpts, ans);
 	}, Object.assign(config, require('./utils/initial-config')))
-	.flatMap(newOpts => {
+	.map(function(newOpts) {
```

Comment:
since we used function at reduce, I figured out we should stick with the convention